editor/src/components/inspector/inspector-common.ts

```diff
@@ -515,9 +516,9 @@ export const nukeAllAbsolutePositioningPropsCommands = (
 }
 
 export type FixedHugFill =
-  | { type: 'fixed'; value: CSSNumber }
-  | { type: 'fill'; value: CSSNumber }
-  | { type: 'hug' }
+  | { type: 'fixed'; value: CSSNumber; source: 'parsed' | 'fallback' }
```

Comment:
Putting the source in here feels very wrong. It might be minor but I think it's worth us creating a separate type that includes the `FixedHugFill` along with the `source`